slide/snowflake_retail_lt.md

```
@@ -160,15 +163,42 @@ marp: true
+
+---
+
+# お客様相談室の高度化
+
+## IVRの改善と高度化
+
+- LLMによる適切な電話転送
+
+---
+
+# ご清聴ありがとうございました。
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)